pkg/manifest/reference.go

```diff
@@ -36,19 +36,39 @@ func PlatformFromEnum(s string) string {
 }
 
 var (
-	referenceFormat = regexp.MustCompile(`^([\w\d-\._]+):([\w\d-\._]+)/?([\w\d-\._]+)?$`)
+	referenceFormat = regexp.MustCompile(`^([^:]+\:)?([\w\d-\._]+)/?([\w\d-\._]+)?$`)
 )
 
 // ParseReference parses a given text as a Reference.
 func ParseReference(text string) (*Reference, error) {
 	matches := referenceFormat.FindStringSubmatch(text)
 	if len(matches) != 4 {
-		return nil, fmt.Errorf("%q is not a valid GetEnvoy reference. Expected format: <flavor>:<version>[/<platform>]", text)
+		return nil, fmt.Errorf("%q is not a valid GetEnvoy reference. Expected format: [<flavor>:]<version>[/<platform>]", text)
 	}
-	return &Reference{strings.ToLower(matches[1]), strings.ToLower(matches[2]), PlatformFromEnum(matches[3])}, nil
+	flavor := strings.ToLower(matches[1])
+	if flavor == "" {
+		flavor = "standard"
+	} else {
+		flavor = flavor[:len(flavor)-1]
+	}
+	version := strings.ToLower(matches[2])
+	platform := PlatformFromEnum(matches[3])
```

Comment:
I see that `PlatformFromEnum` should be unexported 😄

Reply:
yep especially as we now have one package we don't need this exported